packages/frontend/core/.webpack/runtime-config.ts

```diff
@@ -156,6 +160,10 @@ export function getRuntimeConfig(buildFlags: BuildFlags): RuntimeConfig {
     // environment preset will overwrite current build preset
     // this environment variable is for debug proposes only
     // do not put them into CI
-    ...(process.env.CI ? { allowLocalWorkspace: true } : environmentPreset),
+    ...(process.env.CI ? {} : environmentPreset),
```

Comment:
Can you explain why putting under CI flag does not work for `allowLocalWorkspace`?